### DISCUSSION TITLE
[Question][Hw3] Error in make command

### DISCUSSION THREAD

**Discussion by SangHyeon-Lee** [prosyslab-classroom/cs348-information-security#535] [Q&A]
When I try to run "make" under the root directory, I got this error.

make: dune: Command not found
Makefile:8: recipe for target 'all' failed
make: *** [all] Error 127

I installed all the libraries (including dune) and set the environment as the document.
Why does this happen??

Thank you in advance!

**Comment by hieudz** [comment]
Have you tried eval $(opam env)

**Comment by SangHyeon-Lee** [comment]
Oh, it solved it!
Thanks a lot
